Add devnote on missing config files

isilher opened the pull request:
As requested in #48 . Let me know if you would like me to add or change anything!

maxkomarychev commented:
the content is great, thank you @isilher ! one thing though - can you please move it into [the cookbook](https://github.com/maxkomarychev/react-native-ultimate-config/blob/master/docs/cookbook.md) ?

thanks!

maxkomarychev commented:
maybe we also need some general purpose "troubleshooting" section 🤔

maxkomarychev commented on docs/devnotes.md:
```diff
@@ -18,3 +18,55 @@ git update-index --assume-unchanged ios/ConfigValues.h
+
+{
+  "scripts": {
+    "clean": "rm -rf node_modules && yarn && yarn rnuc ./env.yaml"
```
this script makes an assumption about name and format of the env file - not sure if this should be hardcoded here, what do you think?

isilher replied:
True, I explicitly did not use that in the leading documentation. My assumption was that in an example it could help clarify the syntax of the command without implying this should be the name or format used. If you prefer I can change it?

isilher commented:
I moved the notes to a new Troubleshooting docs section and added that to the README indexation.
I also updated the examples to consistently refer to `<env file>` instead of a specific file name and format.

Let me know if this works for you :-)

maxkomarychev commented:
nice, thanks!
